packages/frontend-app/src/app/components/Card/index.scss

```
@@ -0,0 +1,14 @@
+@import "tailwindcss/components";
+@import "tailwindcss/base";
+@import "tailwindcss/utilities";
+
+.card__container {
+  @apply flex flex-col cursor-pointer w-full;
+  & .overlay {
+    @apply absolute bottom-0 left-0 w-full h-full bg-blackAlpha30 flex items-end justify-between p-4 opacity-0 hover:opacity-100 ease-in duration-300;
```

Nitpick: It would have been good to break this into 2/3 applies as this is too long based on scope. This way its more readable

```suggestion
    @apply absolute bottom-0 left-0 w-full h-full bg-blackAlpha30;
    @apply flex items-end justify-between p-4 opacity-0;
    @apply hover:opacity-100 ease-in duration-300;
```